stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
+# You should not edit this file by hand.
+# For more information, please see the documentation at:
+#   https://docs.haskellstack.org/en/stable/lock_files
+
+packages: []
+snapshots:
+- completed:
+    size: 524804
+    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/14/16.yaml
+    sha256: 4d1519a4372d051d47a5eae2241cf3fb54e113d7475f89707ddb6ec06add2888
+  original: lts-14.16
```

Comment:
I don't know why this is in the repository; I don't normally include stack.yaml.lock, but I'm not sure what the best practice is here.  Maybe deleting it would be sensible.

Reply:
I see now that this is an added file in this PR.  It should not be part of the PR.

Reply:
my mistake, removed